src/redux/gjeldendeBruker/types.ts

```diff
@@ -0,0 +1,16 @@
+import { Action } from 'redux';
+
+export interface GjeldendeBrukerState {
+    fødselsnummer: string;
+}
+
+export enum SetNyGjeldendeBrukerActionTypes {
+    SetNyPerson = 'SetNyPerson'
```

Comment:
"GJELDENDEBRUKER / SET"  for å opprettholde actionstil=

Reply:
on it

Reply:
jeg bare tenkte, jaja, jeg skriver noe her, så kommer jørund med riktig navn etterpå :-P